templates/zerver/help/view-all-bots-in-your-organization.md

```diff
@@ -2,7 +2,12 @@
 
 {!admin-only.md!}
 
-You can view a list of all bots in your organization, active or deactivated
-alike, by visiting the [Bots](/#organization/bot-list-admin) {!admin.md!}
-You can [deactivate or reactivate](/help/deactivate-or-reactivate-a-bot) any
-bots in your organization from this panel.
+You might want to see all of the bots in your organization, active or
+deactivated alike, so that you know what kind of automated interactions are
+happening or available to happen on your chat server.
+
+{!go-to-organization-settings.md!}
+
+1. On the left, click on **Bots**. You should see a list of all of the bots in
```

Comment:
I would keep this totally standard. So
```
1. On the left, click on **Bots**.
```
And then in the intro we can talk about deactivate and reactivate, or in a tip, or whatever.

The reason for following the form closely is that eventually we'll want to style this, give instructions conditionally based on ios or android, paid or free tier, if you're an admin or non-admin, etc. Any article that's not to form is something that will likely either break or will generate additional work later.